.markdown_link_check.json

```diff
@@ -18,5 +18,6 @@
     {
       "pattern": "^https://stackoverflow.com/"
     }
-  ]
+  ],
+  "aliveStatusCodes": [200, 206, 300, 301, 302, 303, 304, 307, 308, 429]
```

Comment:
I'm not sure it's a good idea to include the redirection (30x) codes: they tell you what to do to get to the content, e.g. check another URL, but that doesn't mean that action doesn't lead to a dead link.

Regarding 429, I'm not sure, again, that getting this response indicates that the link is not dead. My reading of [the relevant section of RFC6585](https://datatracker.ietf.org/doc/html/rfc6585#section-4) tells me this could be implemented as a response _before_ the HTTP server even decides if the content exists at all.